app/views/overview/_service-bindings.html

```diff
@@ -8,6 +8,15 @@
     secrets="$ctrl.secrets">
   </overview-service-binding>
   <div ng-if="($ctrl.bindableServiceInstances | size) && ({resource: 'bindings', group: 'servicecatalog.k8s.io'} | canI : 'create')">
-    <a href="" ng-click="$ctrl.createBinding()" role="button">Create Binding</a>
+    <a href="" ng-click="$ctrl.createBinding()" role="button">
+      <span class="pficon pficon-add-circle-o"></span>
+      Create Binding
+    </a>
+  </div>
+  <div ng-if="!($ctrl.bindableServiceInstances | size)">
```

Comment:
On the overview page, the service bindings section is not shown at all when there are no service bindings. The create action is in the kebab.